tests/network/vmi_networking.go

```diff
@@ -766,20 +781,32 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 				tcpPort := 8080
 				tests.StartPythonHttpServer(serverVMI, tcpPort)
 
-				assert.XFail("https://github.com/kubevirt/kubevirt/issues/5113", func() {
-					By("Checking ping (IPv6) from vmi to cluster nodes gateway")
-					// Cluster nodes subnet (docker network gateway)
-					// Docker network subnet cidr definition:
-					// https://github.com/kubevirt/project-infra/blob/master/github/ci/shared-deployments/files/docker-daemon-mirror.conf#L5
-					Expect(libnet.PingFromVMConsole(serverVMI, "2001:db8:1::1")).To(Succeed())
-				})
-
 				Expect(verifyClientServerConnectivity(clientVMI, serverVMI, tcpPort, k8sv1.IPv6Protocol)).To(Succeed())
 			},
 				table.Entry("with a specific port number [IPv6]", []v1.Port{{Name: "http", Port: 8080}}, ""),
 				table.Entry("without a specific port number [IPv6]", []v1.Port{}, ""),
 				table.Entry("with custom CIDR [IPv6]", []v1.Port{}, "fd10:10:10::/120"),
 			)
+
+			It("[outside_connectivity]should be able to reach the outside world [IPv6]", func() {
+				ipv6Address := "2001:db8:1::1"
```

Comment:
what about putting those default values as part of the parser ?
(you can use a const, and then put it in the flags parser)
and then we can omit the `if flags...`
but the bash must not send the value in case its `""` if we choose this direction

Reply:
I want to have the same implantation also when we ran the tests without our scripts. For exapmle when running with `go` command or with `sonobouy` . so I don't want to build on the scripts parser fot that. 
besides i Don't think the `if` is that bad.